Makefile.toml

```diff
@@ -100,32 +100,26 @@ rm -rf ${DOCS_FOLDER}/*
 dependencies = ["docs-clean", "schema"]
 description = "Generate documentation"
 script = '''
-echo "🔎 Checking jsonschema2md installed..."
-if ! which jsonschema2md >/dev/null;
-then
-    echo "\n❌ jsonschema2md could not be found"
-    echo "  Consider installing jsonschema2md to generate documentation.\n"
-    echo "＞ \`npm install -g @adobe/jsonschema2md\`\n\n"
-    exit 1
-fi
+echo "📦 Installing fadroma-schema..."
+
+npm install --save-dev @fadroma/schema@1.0.5
```

Comment:
It's not recommended to install a node package without `package.json` file. If you intend to have it readily available, consider a global installation, I mean:

```
npm install -g @fadroma/schema@1.0.5
```

but if you just want to run a package, IMHO, it's best to use [npx](https://www.npmjs.com/package/npx), and a one-off invocation without local installation.

Reply:
Yes, I know but the global installation is not working with this package, maybe an issue on it ? (@egasimus ?)

I will try with npx.

Reply:
With global install or through npx, I have the following error : 

```bash
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/benjamin/Documents/projects/contracts/node_modules/@fadroma/schema/schema.mjs'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

😭

Reply:
Which `node` version do you use ?

```
➜ node --version
v16.18.1

➜ npx @fadroma/schema@1.0.5
Fadroma Schema Renderer (https://fadroma.tech)
Usage:
  node ../../../node_modules/@fadroma/schema/schema.mjs CONTRACT_SCHEMA.json
```

but it makes me think that it's not great to have a tool whose execution depends on specific versions of its environment...

Reply:
@ccamel With the same version of node than your, it's not working for me 😭.

@egasimus Wonderful ! With the 1.0.6 it's working ! Thanks